### PR TITLE
boops: init at 1.4.0

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/boops/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromGitHub, xorg, cairo, lv2, libsndfile, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "BOops";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/boops/default.nix:4]:
```suggestion
  pname = "boops";
```

File: pkgs/applications/audio/boops/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "sjaehn";
+    repo = pname;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/boops/default.nix:9]:
```suggestion
    repo = "BOops";
```

Comment (magnetophon) [reply pkgs/applications/audio/boops/default.nix:9]: Most of the other plugins in this series have the same capitalization:
https://github.com/NixOS/nixpkgs/blob/7ecb53ac5b2fa50eedb2f56310ce1a83ee6aa19e/pkgs/applications/audio/bslizr/default.nix#L4
https://github.com/NixOS/nixpkgs/blob/8284fc30c84ea47e63209d1a892aca1dfcd6bdf3/pkgs/applications/audio/bsequencer/default.nix#L4
https://github.com/NixOS/nixpkgs/blob/8284fc30c84ea47e63209d1a892aca1dfcd6bdf3/pkgs/applications/audio/bshapr/default.nix#L4
https://github.com/NixOS/nixpkgs/blob/8284fc30c84ea47e63209d1a892aca1dfcd6bdf3/pkgs/applications/audio/bjumblr/default.nix#L4

Comment (SuperSandro2000) [reply pkgs/applications/audio/boops/default.nix:9]: Which is also not quite right because pname should match attr name in most cases.

**Pull request merged**